blocks/math.ts

```diff
@@ -54,12 +54,13 @@ export const blocks = createBlockDefinitionsFromJsonArray([
       {
         'type': 'field_dropdown',
         'name': 'OP',
+        'optAriaName': 'Arithmetic operation',
         'options': [
-          ['%{BKY_MATH_ADDITION_SYMBOL}', 'ADD'],
-          ['%{BKY_MATH_SUBTRACTION_SYMBOL}', 'MINUS'],
-          ['%{BKY_MATH_MULTIPLICATION_SYMBOL}', 'MULTIPLY'],
-          ['%{BKY_MATH_DIVISION_SYMBOL}', 'DIVIDE'],
-          ['%{BKY_MATH_POWER_SYMBOL}', 'POWER'],
+          ['%{BKY_MATH_ADDITION_SYMBOL}', 'ADD', 'Plus'],
```

Comment:
NB: Will need be added to messages.

Reply:
Not planning to resolve this as part of this PR--it can be covered with #9301 (if even then since localizing these is likely unimportant for upcoming user tests).